assets/environment-interpreter/activate/activate.d/generate-tcsh-startup-commands.bash

```diff
@@ -76,7 +76,10 @@ generate_tcsh_startup_commands() {
   echo "eval \"\`'$_flox_activations' set-env-dirs --shell tcsh --flox-env '$_FLOX_ENV' --env-dirs \$FLOX_ENV_DIRS:q\`\";"
   echo 'if (! $?MANPATH) setenv MANPATH "empty";'
   echo "eval \"\`'$_flox_activations' fix-paths --shell tcsh --env-dirs \$FLOX_ENV_DIRS:q --path \$PATH:q --manpath \$MANPATH:q\`\";"
-  echo "eval \"\`'$_flox_activations' profile-scripts --shell tcsh --env-dirs \$FLOX_ENV_DIRS:q\`\";"
+  echo 'if (! $?_FLOX_SOURCED_PROFILE_SCRIPTS) set _FLOX_SOURCED_PROFILE_SCRIPTS = "";'
+  # Quote _FLOX_SOURCED_PROFILE_SCRIPTS instead of quoting since tcsh doesn't
```

Comment:
instead of quoting what?

Reply:
I think this comment is on an outdated diff?